packages/orchestration/src/exos/local-orchestration-account.js

```diff
@@ -2,14 +2,17 @@
 import { typedJson } from '@agoric/cosmic-proto/vatsafe';
 import { AmountShape, PaymentShape } from '@agoric/ertp';
 import { makeTracer } from '@agoric/internal';
+import { Shape as NetworkShape } from '@agoric/network';
 import { M } from '@agoric/vat-data';
 import { VowShape } from '@agoric/vow';
-import { heapVowE as E } from '@agoric/vow/vat.js';
 import { TopicsRecordShape } from '@agoric/zoe/src/contractSupport/index.js';
 import { InvitationShape } from '@agoric/zoe/src/typeGuards.js';
+import { E } from '@endo/far';
 import {
   ChainAddressShape,
   ChainAmountShape,
+  DenomAmountShape,
+  DenomShape,
   IBCTransferOptionsShape,
 } from '../typeGuards.js';
```

Comment:
aside: we call them `ThingShape` but put them in files called `typeGuards`. In E, they were called [Guards](http://wiki.erights.org/wiki/Guard-based_auditing). (likewise [guards in Monte](https://monte.readthedocs.io/en/latest/glossary.html#term-guard-coercion)).

Trying it out just to look at it: `DenomGuard` vs. `DenomShape`. Hm. Not much of an improvement.

But our dev audience might already be familiar with the notion by the name "guard":

> The term is used with specific meaning in [APL](https://en.wikipedia.org/wiki/APL_(programming_language)), [Haskell](https://en.wikipedia.org/wiki/Haskell_(programming_language)), [Clean](https://en.wikipedia.org/wiki/Clean_programming_language), [Erlang](https://en.wikipedia.org/wiki/Erlang_programming_language), [occam](https://en.wikipedia.org/wiki/Occam_(programming_language)), [Promela](https://en.wikipedia.org/wiki/Promela), [OCaml](https://en.wikipedia.org/wiki/OCaml), [Swift](https://en.wikipedia.org/wiki/Swift_(programming_language)),[[2]](https://en.wikipedia.org/wiki/Guard_(computer_science)#cite_note-2) [Python](https://en.wikipedia.org/wiki/Python_(programming_language)) from version 3.10, and [Scala](https://en.wikipedia.org/wiki/Scala_(programming_language)) programming languages.
 -- [Guard \(computer science\) \- Wikipedia](https://en.wikipedia.org/wiki/Guard_(computer_science))